metrics/network/remote_network/README.md

```diff
@@ -4,18 +4,20 @@
 `iperf3`using a remote setup. Host A will run a container that will act as a server while host B will run a
 container that will act as a client. The network bandwidth will be measured across the containers.
 
-- The `remote-networking-memory-smem.sh` script measures RSS memory with `smem` tool. Simultaneously,
+- The `remote-networking-memory-smem.sh` script measures RSS and PSS memory with `smem` tool. Simultaneously,
 the script uses a remote setup to run a bandwidth network test.
 
 - The `remote-networking-cpu.sh` script measures the percentage of CPU consumption used while running
 the maximum network bandwidth with `iperf3`.
 
+- The `remote-networking-qperf.sh` script measures TCP and UDP latency with `qperf` tool.
+
 ## Prerequisite
 
 An automatic login from host A (user A) to host B (user B) is needed. You must setup authentication 
 keys to do an ssh login without password.
-The `remote-networking-iperf3.sh`, `remote-networking-memory-smem.sh`, and `remote-networking-cpu.sh`
-scripts are only compatible with `docker 1.12.1`.
+The `remote-networking-iperf3.sh`, `remote-networking-memory-smem.sh`, `remote-networking-qperf.sh`
```

Comment:
Again, this probably isn't strictly required for this PR, but this list is a little difficult to read. I'd be tempted to reformat it to something like the following (but let's see what @klynnrif thinks first):

> The following script are only compatible with `docker 1.12.1`:
> 
> - `remote-networking-cpu.sh`
> - `remote-networking-iperf3.sh`
> - `remote-networking-memory-smem.sh`
> - `remote-networking-qperf.sh`

Reply:
+1